### DISCUSSION TITLE
How to set up a loop in playtime

### DISCUSSION THREAD

**Discussion by sdfalk** [helgoboss/helgobox#1175] [Q&A]
I searched the forum and perhaps missed something, but im just trying to understand if I can set loop length in playtime (1 bar 2 bar 5 bar etc) or does it always have to be set in reaper.

thanks

**Comment by helgoboss** [comment]
Loop length for what? Do you want to play just a part of a clip?

**Comment by sdfalk** [comment]
I want to record a loop at whatever length..2, 4, 8 bars whatever , and just want to know if I can set that up in playtime , or does it always have to be done within reaper.

https://github.com/user-attachments/assets/60bfd53a-61f6-42bf-ac76-545497c1906d

**Comment by helgoboss** [comment]
You mean like this? https://youtu.be/0PIjJlWtciw?si=JXYILTO_0cjdKWkU&t=470

**Reply by sdfalk666** [reply]
Thats it. thanks

If I could offer a suggestion, why not add a contextual menu (as per the uploaded video) for each slot
were you can set loop length. Just a thought

https://github.com/user-attachments/assets/a8520cc6-94f9-4c0b-a334-0abe594368b0

**Reply by helgoboss** [reply]
I could add such a context menu to the "Smart record" button at the top. Having it in the slot menu would be irritating, as this is a global setting, not a per-slot setting. Feel free to create a feature request (= discussion in the "Ideas" category).
